### PR TITLE
[Backport 2023.02.xx] Update MS submodule to include Phase IV and #679

### PR DESCRIPTION
**Pull request by dsuren1**
### Description
This PR updates MS submodules to include Phase IV issues and #679

### Issues
- https://github.com/georchestra/mapstore2-georchestra/issues/647
- https://github.com/georchestra/mapstore2-georchestra/issues/682
- https://github.com/georchestra/mapstore2-georchestra/issues/646
- https://github.com/georchestra/mapstore2-georchestra/issues/644
- https://github.com/georchestra/mapstore2-georchestra/issues/645
- https://github.com/georchestra/mapstore2-georchestra/issues/681
- https://github.com/georchestra/mapstore2-georchestra/issues/679

### CODE REVIEW

File: configs/config.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "map": {
-    "projection": "EPSG:3857",
+    "projection": "EPSG:900913",
```
Comment (tdipisa) [inline configs/config.json:3]: @dsuren1 where this change come from? Correct me if I'm wrong, but it doesn't seem related to any of the issues listed in the PR description and it is not present even in master branch.

**Review by tdipisa — CHANGES_REQUESTED** [review]

File: configs/config.json (hunk 1 above)
Comment (dsuren1) [reply configs/config.json:3]: I observed this [issue](https://github.com/georchestra/mapstore2-georchestra/issues/681) still occurs when EPSG:900913 & EPSG:3857 are interchangeably used. However, georchestra’s other supported projection lists are not affected, hence modifying the default config projection is enough to fix. 
**Note**: We need this on georchestra master too. I will create one.

Comment (landryb) [reply configs/config.json:3]: im not a projection expert and im pretty sure you guys well know that, but from my understanding the usage of the unofficial EPSG:900913 was discouraged in favor of the official EPSG:3857 since 2009..

cf:
- https://epsg.io/900913
- https://wiki.openstreetmap.org/wiki/Web_Mercator
- https://en.wikipedia.org/wiki/Web_Mercator_projection#Identifiers

can we avoid this ?

Comment (dsuren1) [reply configs/config.json:3]: Sure. This seems to be caused when transforming a polygon into a circle when the circle is geodesic. Since few projections seem to work fine because of the minuscule difference in radius, it was overlooked. Will address it.

Comment (tdipisa) [reply configs/config.json:3]: @dsuren1 we have to review the fix on the MS side for this, of course.

Comment (dsuren1) [reply configs/config.json]: PR with fix added on [MS](https://github.com/geosolutions-it/MapStore2/pull/9938). The config change reverted. So the rest can be reviewed. Thanks

**Review by tdipisa — CHANGES_REQUESTED** [review]
@dsuren1 once tested again on the MS side please remember to update the revision

**Comment by dsuren1** [comment]
Ready for re-review after testing this fix in [master](https://github.com/georchestra/mapstore2-georchestra/pull/696)

**Review by tdipisa — APPROVED** [review]
